obal/__init__.py

```diff
@@ -103,6 +127,9 @@ def generate_ansible_args(inventory_path, playbook_path, args):
         ansible_args.append("-%s" % str("v" * args.verbose))
     for extra_var in args.extra_vars:
         ansible_args.extend(["-e", extra_var])
+    for key, value in vars(args).items():
```

Comment:
would it be easier/nicer to use a custom [Action](https://docs.python.org/3/library/argparse.html#argparse.Action) to store this?